apps/electron-app/src/renderer/src/pages/onboarding/OnboardingPage3D.tsx

```
@@ -0,0 +1,956 @@
+
+  useEffect(() => {
+    const loader = new THREE.TextureLoader();
+    loader.load(noiseImage, loadedTexture => {
+      loadedTexture.wrapS = THREE.ClampToEdgeWrapping;
+      loadedTexture.wrapT = THREE.ClampToEdgeWrapping;
+      setTexture(loadedTexture);
+    });
+  }, []);
+
+  useFrame(state => {
+    if (meshRef.current && meshRef.current.material) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)